### PR TITLE
removed files from unsupported ports, i.e. Maemo and WebOS

### PR DESCRIPTION
**Pull request by hoehnp**
When working on implementing a new download mechanism for maps, I noticed that we still have code for the currently unsupported and outdated platforms Maemo and WebOS in our repository. In order to clean up code, I suggest to remove this probably unmaintained code.

### CODE REVIEW

**Comment by metalstrolch** [comment]
From my side good to go, I don't need those and as you stated they are probably unmaintained. Vote for delete.

**Review by metalstrolch — APPROVED** [review]

**Comment by hoehnp** [comment]
Additionally I would also suggest to update the wiki to reflect that some of the platforms mentioned there are actually no longer actively maintained.

**Review by jkoan — CHANGES_REQUESTED** [review]
autoload/osso should probably be deleted as well. For sdl we could probably clean up the event System as well

File: CMakeLists.txt [hunk 1]
```diff
@@ -114,7 +114,6 @@ add_module(gui/gtk "GTK libs not found" FALSE)
 add_module(vehicle/geoclue "geoclue lib not found" FALSE)
 add_module(vehicle/gpsd "gpsd lib not found" FALSE)
 add_module(vehicle/gypsy "gypsy lib not found" FALSE)
-add_module(vehicle/maemo "Default" FALSE)
```
[the remark this answers is not in the document]
Comment (hoehnp) [reply CMakeLists.txt:117]: removed the lines, as requested.

**Review by jkoan — APPROVED** [review]

**Pull request merged**